.github/workflows/deploy.yml

```diff
@@ -7,9 +7,20 @@ on:
 
 jobs:
   fastn-build:
+    permissions: write-all
     runs-on: ubuntu-latest
     steps:
-      - name: Install fastn
-        run: source <(curl -fsSL https://fastn.com/install.sh)
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Pull the latest commits
+        run: git pull
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - run: sh -c "$(curl -fsSL https://fastn.com/install.sh)"
```

Comment:
Why is this command different from `fastn.com/install/`? We are telling the world to use one command and we are privately using a different command, why?